src/3d/qgsrubberband3d.cpp

```diff
@@ -187,7 +228,6 @@ void QgsRubberBand3D::updateGeometry()
 
 void QgsRubberBand3D::updateMarkerMaterial()
 {
-  delete mMarkerMaterial;
```

Comment:
why not delete the marker material anymore?

Reply:
`mMarkerMaterial` is added as a component to `mMarkerEntity`.
The docs say that is component has no parent, ownership is transferred, so we shouldn't be deleting it